lib/trackler/track.rb

```diff
@@ -116,19 +115,11 @@ def dir
     end
 
     def hints
-      if File.exist?(track_hints_filename)
-        File.read(track_hints_filename)
-      else
-        ""
-      end
+      docfile = DocFile.find(basename: 'EXERCISE_README_INSERT', track_dir: dir).render
```

Comment:
This doesn't take into account the image_path, which I think is fine, since the image_path stuff is for the documentation that gets rendered on the site with images from the API. We don't have support for images in the exercise READMEs.